Packs/CommonScripts/Scripts/GenericPollingScheduledTask/GenericPollingScheduledTask.js

```diff
@@ -77,6 +98,9 @@ function setNextRun(ids, playbookId, pollingCommand, pollingCommandArgName, pend
     if (extractMode !== undefined) {
         cmd += ' extractMode="' + extractMode + '" auto-extract="' + extractMode + '"';
     }
+
+    checkCommandSanitized(cmd)
+
     return executeCommand("ScheduleCommand", {
```

Comment:
there is also an executeCommand in `genericPollingScheduled`

Reply:
[Done](https://github.com/demisto/content/pull/33368/commits/32c06538c26b2338b23bdc139007b5f546d27b2d)